main.js

```diff
@@ -116,6 +117,9 @@ function createJitsiMeetWindow() {
     jitsiMeetWindow.on('closed', () => {
         jitsiMeetWindow = null;
     });
+    jitsiMeetWindow.once('ready-to-show', () => {
+        jitsiMeetWindow.show();
```

Comment:
This is a common practice in Electron apps.
There is a minor time difference in rendering of the app. During that time a white-ish blank screen is shown. 
This time is larger in slow systems. This will fix that.

Reply:
Excellent, thank you Akshit!